chaoslib/litmus/service_delete.yml

```diff
@@ -0,0 +1,31 @@
+- name: Get the service
+  k8s_facts:
+    kind: Service
+    namespace: "{{ s_ns }}"
```

Comment:
A very minor comment, open for feedback.  

Across the libs, we are looking at keeping the var names consistent. `{{ a_ns }}` is what is being used to filter the app's namespace via the standard vars step: `a_ns: "{{ lookup('env','APP_NAMESPACE') }}"`. 
With this, only the vars specific to that chaoslib are being passed explicitly via include vars.  

The service namespace is essentially the same as the app namespace. Can we reuse that? There could be cases where service kills are effected at one ns, but a check is conducted against an app (like a liveness client) in another namespace, in which case we could extend the convention to `<type>_a_ns`.

Reply:
I was thinking that `s_ns` would be an acronym for service namespace. But as you said, i will rename it to `a_ns` to keep it consistent with other experiments.

Reply:
So what about `s_name` ? Do you think it should be `a_name` ?

Reply:
`s_name` should still be fine @kmjayadeep ! It allows for the possibility of an experiment job to take the service name of the app as an ENV along w/ the regular info (label, ns)